### PR TITLE
added comments

### PR DESCRIPTION
**Pull request by anup-shinde**
Please review my changes

### CODE REVIEW

File: python.txt [hunk 1]
```diff
@@ -7,4 +7,7 @@ def addition(a,b):
 
 def substraction(a,b):
  	sub = a-b
-	return sub
\ No newline at end of file
+	return sub
+
+#this is my changes
```
Comment (akki-9696) [inline python.txt:12]: what is this change

Comment (anup-shinde) [reply python.txt:12]: this is comment

Comment (akki-9696) [reply python.txt:12]: okok

**Review by akki-9696 — APPROVED** [review]
approved

**Pull request merged**